gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -174,18 +170,18 @@ def _get_alts_pysam(self, alts):
       result['id'] = item[0]
       result['desc'] = item[1]['Description'].strip("\"")
```

Comment:
We sometimes we need to perform `strip("\"")`? I saw it a few other times in this module?
Is it just because of passing our integration tests? If that's the case we better modify our tests and remove all these extra operations.
Not needed for this PR, I added an item to [the issue](#534) that tracks these future clean ups.

Reply:
No, it's because Header fields have for strings come with quotes (ie. Description="Some Description"), this operation is very much needed.

Record gets everything after '=', so we get record['Description']='"SomeDescription"'.

Also, I verified, VCF file format says that they specifically should be double quotes, never single quotes.